packages/react-ui-kit/src/Form/Select.tsx

```diff
@@ -112,16 +124,35 @@ const dropdownStyles = (theme: Theme, isDropdownOpen: boolean): CSSObject => ({
   zIndex: 9,
 });
 
-const dropdownOptionStyles = (theme: Theme, isSelected: boolean): CSSObject => ({
+const dropdownOptionStyles = (
+  theme: Theme,
+  isSelected = false,
+  isMultiSelect = false,
+  isDisabled = false,
+): CSSObject => ({
   background: isSelected ? theme.general.primaryColor : theme.general.backgroundColor,
   listStyle: 'none',
   padding: '10px 20px 14px',
-  cursor: 'pointer',
+  cursor: isDisabled ? 'not-allowed' : 'pointer',
   fontSize: '16px',
   fontWeight: 300,
   lineHeight: '24px',
   letterSpacing: '0.05px',
   color: isSelected ? theme.Select.contrastTextColor : theme.general.color,
+  ...(isMultiSelect
+    ? {
+        display: 'grid',
+        gridTemplateAreas: `"checkbox label"
+      ". description"`,
+        gridTemplateColumns: '30px 1fr',
+      }
+    : {
+        '&:hover, &:active, &:focus': {
+          background: theme.general.primaryColor,
+          borderColor: theme.general.primaryColor,
+          color: theme.Select.contrastTextColor,
+        },
+      }),
   '&:first-of-type': {
     borderRadius: '10px 10px 0 0',
```

Comment:
```suggestion
    borderRadius: '12px 12px 0 0',
```
Could you adress this feedback from Anette? (also in lines 116 and 160)
![image](https://user-images.githubusercontent.com/78490891/178452847-b22bba37-190f-4a52-b4e1-443958e37f2d.png)